src/components/NoProgram/NoProgram.jsx

```diff
@@ -0,0 +1,14 @@
+import Section from '../Section/Section'
+import css from './NoProgram.module.css'
+
+const NoProgram = () => {
+  return (
+    <Section sectionClass={css.noProgram}>
+      <h1 className={css.noProgramTitle}>Program</h1>
```

Comment:
To nie będzie h1. `<h1>` może być tylko jedno na stronie i u nas jest w Bannerze :) Zerknij sobie w komponent Section w którym jest props title, użyj go zamiast tego :)